src/components/CollectionsAndPalettes/Collections/Collection/Collection.tsx

```diff
@@ -34,81 +43,69 @@ class Collection extends React.Component<Props, any> {
-          title1: "Living Room",
-          title2: "Rhonda Roomdesigner",
-          title3: "Scandinavian Oasis",
-          price: null,
-          imagePath:
-            "https://www.mydomaine.com/thmb/MNBaDGmg4IW7tOvl3pxVNpqQ6uE=/2500x3049/filters:fill(auto,1)/DesignbyEmilyHendersonDesignandPhotobySaraLigorria-Tramp_654-b8122ec9f66b4c69a068859958d8db37.jpg",
-        },
-
-        {
-          type: "palette",
-          title1: "Palette",
-          title2: "David Designername",
-          title3: "Textils",
-          price: null,
-          imagePath:
-            "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS5brHIzPOkAv7A4E5ul_mT5BaCRbykzf1xvA&usqp=CAU",
-        },
+      items: [
         {
-          type: "sample",
-          title1: "Indusparquet",
-          title2: "Brazilian Pecan Engineered",
-          title3: "$8.95 / sq ft",
-          price: "6",
-          imagePath:
-            "https://www.haro.com/media/custom/produktfinder/parkett/draufsicht/792x865/535575_HARO_PARKETT_Schiffsboden_Wenge_Favorit_Ver.jpg",
+          id: 1,
+          collectionId: 58,
+          position: 1,
+          name: "nume",
+          objectId: "1",
+          objectType: "upload",
+          upload: {
+            name: "some_name",
+            s3Url:
+              "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS5brHIzPOkAv7A4E5ul_mT5BaCRbykzf1xvA&usqp=CAU",
+          },
+          updatedBy: 92,
+          createdOn: "createdon",
+          updatedOn: "updatedon",
         },
         {
-          type: "room",
-          title1: "Indusparquet",
-          title2: "Brazilian Pecan Engineered",
-          title3: "$8.95 / sq ft",
-          price: "6",
-          imagePath:
-            "https://www.john-lewis.co.uk/wp-content/uploads/2018/09/JLH-10.1443821-2500x3559.jpg",
+          id: 2,
+          collectionId: 56,
+          position: 2,
+          name: "nume2",
+          objectId: "2",
+          objectType: "upload",
+          upload: {
+            name: "some_name2",
+            s3Url:
+              "https://www.mydomaine.com/thmb/MNBaDGmg4IW7tOvl3pxVNpqQ6uE=/2500x3049/filters:fill(auto,1)/DesignbyEmilyHendersonDesignandPhotobySaraLigorria-Tramp_654-b8122ec9f66b4c69a068859958d8db37.jpg",
+          },
+          updatedBy: 92,
+          createdOn: "createdon2",
+          updatedOn: "updatedon2",
         },
         {
-          type: "room",
-          title1: "Indusparquet",
-          title2: "Brazilian Pecan Engineered",
-          title3: "$8.95 / sq ft",
-          price: "6",
-          imagePath:
-            "https://www.thespruce.com/thmb/ygpyRaJpg4ubo3l-sDmiQph9YuQ=/1500x1000/filters:no_upscale():max_bytes(150000):strip_icc()/Scandi1-590ba2563df78c9283f4febf.jpg",
+          id: 3,
+          collectionId: 56,
+          position: 3,
+          name: "material",
+          objectId: "3",
+          objectType: "material",
+          material: {
+            id: 100306199,
```

Comment:
We should add a new sku field (string) to the MaterialT and use that instead of the id. That's most likely how it's going to be on the API when it gets added.